tests/standalone_tests/test_cleanup_after_algorithm_execution.py

```diff
@@ -916,31 +960,35 @@ async def test_cleanup_controller_restart(
     localnode1_tables_after_cleanup = localnode1_tasks_handler.get_tables(
         request_id=request_id, context_id=context_id
     )
-    localnodetmp_tables_after_cleanup = localnodetmp_tasks_handler.get_tables(
+    localnode2_tables_after_cleanup = localnode2_tasks_handler.get_tables(
         request_id=request_id, context_id=context_id
     )
 
     start = time.time()
     while (
         globalnode_tables_after_cleanup
         or localnode1_tables_after_cleanup
-        or localnodetmp_tables_after_cleanup
+        or localnode2_tables_after_cleanup
     ):
+        node_lanadscape_aggregator._update()
+
         globalnode_tables_after_cleanup = globalnode_tasks_handler.get_tables(
             request_id=request_id, context_id=context_id
         )
         localnode1_tables_after_cleanup = localnode1_tasks_handler.get_tables(
             request_id=request_id, context_id=context_id
         )
-        localnodetmp_tables_after_cleanup = localnodetmp_tasks_handler.get_tables(
+        localnode2_tables_after_cleanup = localnode2_tasks_handler.get_tables(
             request_id=request_id, context_id=context_id
         )
         now = time.time()
         if now - start > WAIT_CLEANUP_TIME_LIMIT:
             pytest.fail(
-                f"Some of the nodes were not cleaned during {WAIT_CLEANUP_TIME_LIMIT=}"
+                f"Some of the nodes were not cleaned during {WAIT_CLEANUP_TIME_LIMIT=}\n"
+                f"{globalnode_tables_after_cleanup=}\n{localnode1_tables_after_cleanup=}\n"
+                f"{localnodetmp_tables_after_cleanup=}"
             )
-        time.sleep(1)
+        time.sleep(0.5)
 
     controller.stop_node_landscape_aggregator()
```

Comment:
I think this is not needed anymore since the aggregator is never started.

Reply:
yes, removed